### PR TITLE
DreamSub Discord Presence

### PR DESCRIPTION
**Pull request by diskxo**
I created the discord presence for the website dreamsub.stream.
This is an example of the final presence:
https://prnt.sc/v3eqww

### CODE REVIEW

File: websites/D/DreamSub/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../../tsconfig.json",
+
+  "compilerOptions": {
+    "outDir": "./dist/"
+
+  }
+
+}
```
Comment (Bas950) [inline websites/D/DreamSub/tsconfig.json:9]: fix spacing

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix codacy issues.

**Comment by diskxo** [comment]
Ok, I should have fixed everything

**Review by Bas950 — APPROVED** [review]
Fix codacy issues.

**Review by ririxichan — APPROVED** [review]

**Pull request merged**